README.md

```diff
@@ -1,8 +1,130 @@
 # cozy-client
+
 A simple and declarative way of managing [cozy-stack](https://github.com/cozy/cozy-stack) API calls and resulting data.
 
 ## Introduction
+
 `cozy-client` is a convenient yet powerful way to bind `cozy-stack` queries to your (p)React components.
 
 If you don't use React, you can still benefit of its underlying libraries:
- - [cozy-stack-link](https://github.com/cozy/cozy-client/blob/master/packages/cozy-stack-link/README.md): a HTTP client to the cozy stack.
\ No newline at end of file
+
+* [cozy-stack-link](https://github.com/cozy/cozy-client/blob/master/packages/cozy-stack-link/README.md): a HTTP client to the cozy stack.
+
+## Setup
+
+### Install
+
+`npm install --save cozy-client cozy-stack-link`
+or
+`yarn add cozy-client cozy-stack-link`
+
+To get started using `cozy-client` with (p)React, you need to create a `CozyClient`, a `CozyStackLink`, and a `CozyProvider`:
+
+* `CozyClient` is the master of your data: it manages data queries and their status ;
+* `CozyStackLink` is the HTTP interface to the stack instance ;
+* `CozyProvider` injects the client into components' context.
+
+#### Creating a client
+
+```js
+import CozyClient from 'cozy-client'
+import CozyStackLink from 'cozy-stack-link'
+
+// We need to retrieve the domain name of the cozy instance and a token from the HTML
+// See https://cozy.github.io/cozy-docs-v3/en/dev/app/#behind-the-magic
+const root = document.querySelector('[role=application]')
+const data = root.dataset
```

Comment:
I would not add these particular details. Let use magic variable here as the library could be used outside the classical way described here https://cozy.github.io/cozy-docs-v3/en/dev/app/#behind-the-magic

I think that documentation should be necessary and sufficient (as simple and concise as possible) to limit unnecessary knowledge biases.